### PR TITLE
[ES] Add `Storyboard/Scripting/General Rules`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Storyboard/Scripting/General_Rules/es.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Todos los sprites del storyboard se colocan debajo de la skin y los [objetos](/wiki/Gameplay/Hit_object). Por lo tanto, incluso la capa «más alta» (primer plano) en el storyboard siempre estará detrás de la barra de HP, los círculos/sliders/spinners, el cursor, etc.
+
+Estas son las cuatro capas de un storyboard, en orden creciente de prioridad:
+
+- Fondo
+- Fail (solo se muestra si el jugador está en el «estado Fail», consulta [estado de juego](#estado-de-juego) a continuación)
+- Pass (solo se muestra si el jugador está en el «estado Pass», consulta [estado de juego](#estado-de-juego) a continuación)
+- Primer plano
+
+Ten en cuenta que las capas «Fail» y «Pass» nunca aparecen en pantalla simultáneamente, a diferencia de lo que ocurre en la pestaña de diseño.
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/Storyboard/Scripting/General_Rules/es.md:45]:
```suggestion
Ten en cuenta que las capas «Fail» y «Pass» nunca aparecen en la pantalla simultáneamente, a diferencia de lo que ocurre en la pestaña de diseño.
```

File: wiki/Storyboard/Scripting/General_Rules/es.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+- De lo contrario, estado Fail. Ten en cuenta que no hay un estado solo para Katu/Beat!, a diferencia de los juegos de DS (que tenían tres estados).
+  - En [osu!taiko](/wiki/Game_mode/osu!taiko), estado Fail si el jugador falló la última nota, estado Pass en caso contrario.
+  - En [osu!catch](/wiki/Game_mode/osu!catch), este es siempre el estado en el que estaba el descanso anterior. La primera sección jugable siempre será estado Pass.
+
+Estados durante el **tiempo de descanso** (entre segmentos de tiempo de juego):
+
+- Estado Pass si la barra de HP terminó por encima de la mitad en la última sección de tiempo de juego (es decir, aparece el símbolo «O»).
+- De lo contrario, estado Fail (es decir, aparece el símbolo «X»).
+  - En [osu!taiko](/wiki/Game_mode/osu!taiko), si se alcanza cierta cuota en un momento determinado. Consulta los dos ejemplos siguientes.
+    - Ejemplo A: Obtener una precisión de 96,5 % mientras la barra de HP todavía está al 40 % dará un Pass en lugar de un Fail.
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/Storyboard/Scripting/General_Rules/es.md:75]:
```suggestion
    - Ejemplo A: Obtener una precisión de 96,5 % mientras la barra de HP todavía está al 40 % dará un Pass en lugar de un Fail.
```

File: wiki/Storyboard/Scripting/General_Rules/es.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    - Ejemplo B: Obtener demasiados 100 en aproximadamente 30 notas y obtener una D mientras la barra de HP todavía está alrededor del 30 % resultará en un Fail en lugar de un Pass (en este caso, consulta [ZUN - Maiden's Cappricio ~ Dream Battle](https://osu.ppy.sh/beatmapsets/18005#taiko/69556)).
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/Storyboard/Scripting/General_Rules/es.md:76]:
```suggestion
    - Ejemplo B: Obtener demasiados 100 en aproximadamente 30 notas y obtener una D mientras la barra de HP todavía está alrededor del 30 % resultará en un Fail en lugar de un Pass (en este caso, consulta [ZUN - Maiden's Cappricio ~ Dream Battle](https://osu.ppy.sh/beatmapsets/18005#taiko/69556)).
```

File: wiki/Storyboard/Scripting/General_Rules/es.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+- El tiempo en el SB no depende de la temporización del beatmap en sí (por ejemplo, cuántos compases hay o pulsos por minuto). Por lo tanto, se recomienda que el beatmap esté razonablemente temporizado antes del storyboarding, ya que será más difícil ajustar estos tiempos más adelante.
+- El tiempo no está limitado a la duración de la canción. Es posible tener valores negativos para eventos antes de que comience la canción (una introducción) y para valores que se extienden más allá de la última sección jugable o incluso del final del archivo de audio (un cierre).
+- Cuando se carga, el beatmap comenzará desde el primer evento especificado o desde el tiempo 0, lo que ocurra primero.
+  - En el primer caso, se mostrará al usuario el botón `Skip`. Al hacer clic en él o al presionar `Barra espaciadora` saltará al tiempo 0. El juego vuelve al comportamiento normal de salto previo al mapa (por ejemplo, presiona `Skip` nuevamente para ir directamente a la cuenta regresiva, a diferencia de [Elite Beat Agents](https://es.wikipedia.org/wiki/Elite_Beat_Agents), donde reiniciar el beatmap lleva al jugador hasta el inicio, no hasta el tiempo 0).
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/Storyboard/Scripting/General_Rules/es.md:95]:
```suggestion
  - En el primer caso, se mostrará al usuario el botón `Skip`. Al hacer clic en él o al presionar `Barra espaciadora` saltará al tiempo 0. El juego vuelve al comportamiento normal de salto previo al mapa (por ejemplo, presionar `Skip` nuevamente para ir directamente a la cuenta regresiva, a diferencia de [Elite Beat Agents](https://es.wikipedia.org/wiki/Elite_Beat_Agents), donde reiniciar el beatmap lleva al jugador hasta el inicio, no hasta el tiempo 0).
```

File: wiki/Storyboard/Scripting/General_Rules/es.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+- El juego pasará a la [pantalla de resultados](/wiki/Client/Interface#pantalla-de-resultados) tan pronto como ocurra el último evento, o el usuario haga clic en el botón `Skip` o presione `Barra espaciadora`.
+  - Esto incluye eventos que se encuentran en **AMBAS** capas de Pass/Fail, aunque solo se mostrará una.
+    - Ejemplo: si el storyboard de Fail termina en el tiempo 20 000 y el storyboard de Pass termina en el tiempo 25 000, el juego esperará hasta el momento 25 000 incluso si el jugador está en estado Fail (todos los objetos desaparecerán). Por lo tanto, es mejor asegurarse de que las variantes de finalización de Pass y Fail tarden la misma cantidad de tiempo en completarse.
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/Storyboard/Scripting/General_Rules/es.md:98]:
```suggestion
    - Ejemplo: si el storyboard de Fail termina en el tiempo 20 000 y el storyboard de Pass termina en el tiempo 25 000, el juego esperará hasta el momento 25 000 incluso si el jugador está en estado Fail (todos los objetos desaparecerán). Por lo tanto, es mejor asegurarse de que las variantes de finalización de Pass y Fail tarden la misma cantidad de tiempo en completarse.
```

**Review by Unnombredeusuarioextremadamentelargo — CHANGES_REQUESTED** [review]

**Review by cl8n — APPROVED** [review]

**Comment by cl8n** [comment]
wont block this one for it but I think if you add new images in a way that they aren't literal translations of the English ones, then English should also be updated

**Pull request merged**